src/test/java/com/verint/todoapi/ToDoControllerTest.java

```diff
@@ -76,4 +80,46 @@ void postToDo_returnsToDoDTO() throws Exception{
                                             """));
     }
 
+    @Test
+    void deleteToDo_callServiceDeleteWithIdGiven() throws Exception{
+        ArgumentCaptor<ToDoDTO> argumentCaptor = ArgumentCaptor.forClass(ToDoDTO.class);
+        when(toDoService.delete(argumentCaptor.capture())).thenReturn(true);
+
+        mockMvc.perform(delete("/todos")
+                .contentType(APPLICATION_JSON)
+                .content("""
+                         {"id":1}
+                        """));
+
+        verify(toDoService).delete(argumentCaptor.capture());
+        assertThat(argumentCaptor.getValue(), is(toDoDTO(1L,null)));
+    }
+
+    @Test
+    void deleteToDo_idInDatabase_returnsSuccessMessage() throws Exception{
+        ArgumentCaptor<ToDoDTO> argumentCaptor = ArgumentCaptor.forClass(ToDoDTO.class);
+        when(toDoService.delete(argumentCaptor.capture())).thenReturn(true);
+
+        mockMvc.perform(delete("/todos")
+                .contentType(APPLICATION_JSON)
+                .content("""
+                         {"id":1}
+                        """))
```

Comment:
This will become redundant when you change to delete with the id being a path variable, but the purpose of the """ is to help with formatting and so align them and the content relative to them.
```suggestion
                .content("""
                           {"id":1}
                         """))
```